tests/unit/specs/components/LicenseCode.spec.js

```diff
@@ -0,0 +1,98 @@
+import { mount, createLocalVue, config } from '@vue/test-utils'
+import LicenseCode from '@/components/LicenseCode.vue'
+import VueI18n from 'vue-i18n'
```

Comment:
I searched on the internet about this warning and found that this occurs when the translation is not found and the `i18n` is using `fallbackLocale` which in our case is `en`. Since we are testing for `en` only there should not be any issues with them
I suppressed then with an attribute